src/Units/AWBWUnits.java

```diff
@@ -0,0 +1,778 @@
+    private static final int IDLE_FUEL_BURN = 5;
+    private static final int MAX_AMMO = 9;
+    private static final int VISION_RANGE = 2;
+    private static final int MOVE_POWER = 9;
+
+    private static final MoveType moveType = new Flight();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new AWBWWeapons.FighterMissiles() };
+
+    public FighterModel()
+    {
+      super("Fighter", AWBWUnitEnum.FIGHTER, UnitRoleEnum.AIR_SUPERIORITY, ChassisEnum.AIR_HIGH, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class StealthModel extends AWBWUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 24000;
+    private static final double STAR_VALUE = 2.0;
+    private static final int MAX_FUEL = 60;
+    private static final int IDLE_FUEL_BURN = 5;
+    private static final int MAX_AMMO = 6;
+    private static final int VISION_RANGE = 4;
+    private static final int MOVE_POWER = 6;
+
+    private static final MoveType moveType = new Flight();
+    private static final UnitActionType[] actions = UnitActionType.COMBAT_VEHICLE_ACTIONS;
+    private static final WeaponModel[] weapons = { new AWBWWeapons.StealthShots() };
+
+    public StealthModel()
+    {
+      super("Stealth", AWBWUnitEnum.STEALTH, UnitRoleEnum.AIR_ASSAULT, ChassisEnum.AIR_HIGH, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+    }
+  }
+
+  public static class StealthHideModel extends StealthModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int IDLE_FUEL_BURN = 8;
+
+    public StealthHideModel()
+    {
+      super();
+      type = AWBWUnitEnum.STEALTH_HIDE;
+      idleFuelBurn = IDLE_FUEL_BURN;
+      hidden = true;
+    }
+  }
+
+  public static class BBombModel extends AWBWUnitModel
+  {
+    private static final long serialVersionUID = 1L;
+    private static final int UNIT_COST = 25000;
+    private static final double STAR_VALUE = 0.6;
+    private static final int MAX_FUEL = 45;
+    private static final int IDLE_FUEL_BURN = 5;
+    private static final int MAX_AMMO = -1;
+    private static final int VISION_RANGE = 1;
+    private static final int MOVE_POWER = 9;
+
+    private static final MoveType moveType = new Flight();
+    private static final UnitActionType[] actions = UnitActionType.BASIC_ACTIONS;
+    private static final WeaponModel[] weapons = {};
+
+    public BBombModel()
+    {
+      super("BBomb", AWBWUnitEnum.BBOMB, UnitRoleEnum.AIR_ASSAULT, ChassisEnum.AIR_HIGH, UNIT_COST, MAX_AMMO, MAX_FUEL, IDLE_FUEL_BURN, VISION_RANGE, MOVE_POWER,
+          moveType, actions, weapons, STAR_VALUE);
+      possibleActions.add(0, new UnitActionType.Explode(5, 3));
```

Comment:
Magic 0. Minor instance, but I will still normally try to use a named temp.

Reply:
It's just slapping it to the start of the array.
If you don't know what index 0 means, you probably shouldn't be messing around in the codebase lol

Reply:
I mean, I know what that line does, I just don't like magic numbers. They are slightly more forgivable on standard library functions I guess, which is why I said it's a "minor instance".

The 5 and 3 are also magic, and that's not a standard library call.

Reply:
Fair enough, I suppose.

You definitely have a point on the Explode constructor there, though.